app/controllers/spree/solidus_six_saferpay/checkout_controller.rb

```diff
@@ -139,20 +138,24 @@ def fail
 
       private
 
-      def initialize_payment(order, payment_method)
-        raise NotImplementedError, "Must be implemented in PaymentPageCheckoutController or TransactionCheckoutController"
+      def initialize_payment(_order, _payment_method)
```

Comment:
I pass them here to document the interface of this method.
Ruby has no real way of doing `abstract` methods, so my preferred way of handling this is to implement the method in the base class to raise an error, and override this method in child classes with child-specific functionality.
This means that the arguments are technically useless in the base class method, but I still want to have them so that one can directly see the method signature and expected arguments when reading the code of the base class (because this is what you read when you are trying to figure out how to implement a child class).